### ISSUE TITLE
I Love you!

### ISSUE THREAD

**Issue by zerokurns** [WolverineFan/YNABLinuxInstall#16]
Not an issue, just a thank-you! I really appreciate the time and effort you put into this script and readme. I've installed YNAB4 on Debian with absolutely no troubles.

Thank you again!

**Comment by stevesbrain** [comment]
Seconded wholeheartedly!

**Comment by strootman** [comment]
Yep! Just applied the update with no issues. Thanks! And great work! Your script is commented so well, I could show it as an example to newbies on what well-commented code looks like.

**Comment by tclemans** [comment]
Installed YNAB4 on Linux Mint 17.1 with no problems. Excellent script. Thanks!

**Comment by MarcPer** [comment]
Yes, very handy.

**Comment by apaolillo** [comment]
Very useful! Thank you :)

**Comment by dlindelof** [comment]
This is a lifesaver for me. Thank you so much for making this available.

**Comment by nevarsin** [comment]
same here, you saved me a lot of time. Thanks a lot.

**Comment by bgigous** [comment]
AMAZING, I LOVE YOU AS WELL! <3
Ahem... I chose option 3 and it worked flawlessly. Great work!

#### Note to Steam Users:

I bought YNAB4 through Steam. I was trying to figure out how to use this script to link YNAB4 via Steam with Linux for Dropbox, but I discovered I didn't have to do that at all. Instead I found my YNAB4 key here:

`/home/user/.wine/drive_c/Program Files (x86)/Steam/steamapps/common/YNAB 4`

Plugged it into the YNAB app that I downloaded and installed through this script and it worked just fine.

**Comment by andrecastillo** [comment]
Jumping in here to say thanks!
